common/membership/interfaces.go

```diff
@@ -23,21 +23,13 @@
 package membership
 
 import (
-	"errors"
-
 	"github.com/uber/cadence/common"
 	"github.com/uber/cadence/common/types"
 )
 
-// ErrUnknownService is thrown for a service that is not tracked by this instance
-var ErrUnknownService = errors.New("Service not tracked by Monitor")
-
 // ErrInsufficientHosts is thrown when there are not enough hosts to serve the request
 var ErrInsufficientHosts = &types.InternalServiceError{Message: "Not enough hosts to serve the request"}
```

Comment:
What is the reason for removing up/below errors but not this one? All three of them does not seem to be used outside of this package.

Reply:
will address this in next PR